### PR TITLE
update versioning

### PR DESCRIPTION
**Pull request by anntisa**
most of this is random changes that were made when android studio tried running on my comp

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -23,13 +23,13 @@ dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     testCompile 'junit:junit:4.12'
     compile 'com.android.support:appcompat-v7:23.2.0'
-    compile 'com.survata.android:library:1.0.12'
+    compile 'com.survata.android:library:1.0.13'
```
Comment (anntisa) [inline app/build.gradle]: change to reflect new SDK version

File: app/build.gradle [hunk 2]
```diff
@@ -23,13 +23,13 @@ dependencies {
...
     compile 'com.android.support:support-v4:23.2.0'
     compile 'com.android.support:preference-v14:23.1.1'
 }
 
 repositories {
     maven {
-        url  "http://dl.bintray.com/theresagao/maven"
+        url  "http://dl.bintray.com/survata/maven"
```
Comment (anntisa) [inline app/build.gradle]: our bintray account

File: build.gradle [hunk 3]
```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.2.0'
+        classpath 'com.android.tools.build:gradle:2.1.3'
```
Comment (anntisa) [inline build.gradle]: android studio auto updated the gradle

File: .idea/encodings.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="Encoding">
+    <file url="PROJECT" charset="UTF-8" />
+  </component>
+</project>
```
Comment (jameswwarner) [inline .idea/encodings.xml]: newline

File: app/build.gradle (hunk 2 above)
Comment (jameswwarner) [inline app/build.gradle]: :+1:

**Comment by jameswwarner** [comment]
One minor thing, otherwise 🚢

**Pull request closed without merging**